node.js

```diff
@@ -82,7 +82,9 @@ Node.prototype.addChild = function (node) {
       return
     }
 
-    node.parametricBrother = parametricBrother
+    if (node !== this) {
+      node.parametricBrother = parametricBrother
+    }
```

Comment:
Can you add a unit test for this change? When would this happen

Reply:
This would happen when `iterate(this)` [is invoked](https://github.com/delvedor/find-my-way/pull/105/files#diff-28a3689be95c88dd5e7a37db516ab084R95). `this` remains constant meaning `node === this` on the first call. We want to ignore that case and just recursively iterate it's children.

Reply:
+1, can you add a unit test, or is it already included?

Reply:
I will try add a test case for that soon :)

Reply:
Test case Added :)